src/components/QueryResult.vue

```diff
@@ -9,14 +9,18 @@
 		<div v-else-if="encodedQuery.length === 0" class="querybuilder__result__description">
 			{{ $i18n('query-builder-result-placeholder')}}
 		</div>
-		<iframe
-			v-else
-			:src="queryServiceEmbedUrl + '#' + encodedQuery"
-			:key="iframeRenderKey"
-			class="querybuilder__result__iframe"
-			referrerpolicy="origin"
-			sandbox="allow-scripts allow-same-origin allow-popups">
-		</iframe>
+		<div v-else class="querybuilder__result__wrapper">
+			<div class="querybuilder__result__header">
+				<h5>{{ $i18n('query-builder-result-header')}}</h5>
```

Comment:
This should be an `<h2>`, though it might need some more styling

Reply:
The figma spec says heading 5, I'm slightly confused about this now.

Reply:
I think that might be referring to a set of styles dubbed "heading 5", but lets ask to be on the safe side